backend/src/models/userModel.ts

```diff
@@ -10,6 +10,8 @@ export interface IUser extends Document {
   age?: number;
   school?: string;
   major?: string;
+  friends: [string],
+  friendRequests: [string]
```

Comment:
Just being nit picky but I _think_ you could change these lines for `string[]` instead of `[string]` and it reads a bit better.